src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Decentralized finance (defi)
+description: An overview of defi on Ethereum
+lang: en
+template: use-cases
+sidebar: true
+image: ../../assets/use-cases/defi.png
+sidebarDepth: 2
+summaryPoints:
+  [
+    "A global, open alternative to the current financial system.",
+    "Probably the biggest use case for Ethereum right now.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+_Defi is an alternative to a financial system that many of us have grown to mistrust thanks to the cycles of mismanagement, crashes, and bail outs. It's an open global system that gives you more control and visibility over your money. It's exposure to global markets when your currency at home is failing. And there are no banker bonuses – it's largely owned by its users or maintained as a public good. And right now there's over 40 billion dollars worth of crypto in defi applications._
+
+## What's defi?
+
+Defi is a collective term for products that are making financial products and services accessible to anyone who can use Ethereum – anyone with an internet connection. With defi, there are no centralized authorities who can block payments or deny you access to anything.
```

Comment:
```suggestion
Defi is a collective term for financial products and services that are accessible to anyone who can use Ethereum – anyone with an internet connection. With defi, there are no centralized authorities who can block payments or deny you access to anything.
```

Reply:
> With defi, there are no centralized authorities who can block payments or deny you access to anything.

I like this but it in addition to focusing on the negative of the current system, perhaps worth touching on the positive aspects / innovation of the new system:
1. automation: programs (that anyone can inspect & scrutinize) that can perform the operations that are currently executed by fallible humans.
2. always on! no arbitrary times when the market is open